definitions/ext-oracle/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: EXT
+type: ORACLE
+synthesis:
+  name: database
+  identifier: database
+  encodeIdentifierInGUID: true
+  conditions:
+  - attribute: event_type
```

Comment:
Shouldn't this be `eventType`?

Reply:
In the Oracle eventType the attribute is called "event_type".
![image](https://user-images.githubusercontent.com/52978309/125085122-3db73600-e098-11eb-8dfc-abb3cd34d79a.png)

Reply:
I think that for the definition it should still be `eventType`

Reply:
Updated. Please check the latest commit.

Reply:
Thanks!